static/bidder-params/inmobi.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "InMobi Adapter Params",
+  "description": "A schema which validates params accepted by the InMobi adapter",
+  "type": "object",
+  "properties": {
+    "plc": {
+      "type": ["string", "integer"],
+      "description": "An ID corresponding to the placement selling the impression"
+    }
+  },
+  "required": ["plc"]
```

Comment:
Please consider adding a check in your adapter for this field being present to avoid a call being made to the InMobi servers which will most certainly fail (since it's required).

Reply:
Added.